reference/extensions/python_api/ConanAPI.rst

```diff
@@ -3,7 +3,7 @@
 Conan API Reference
 ===================
 
-.. include:: api_warning.rst
+.. include:: ../../../common/experimental_warning.rst
```

Comment:
@memsharded  I think it's better to have all of them in one place. Wdyt?

Reply:
Yes, they can be even in the repo root, not a big issue for me.